### PR TITLE
[EX-49] Move `useSimulcast` hook logic to `useCamera` hook in react native webrtc sdk

### PR DESCRIPTION
**Pull request by skyman503**

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -532,6 +545,49 @@ export function useCamera() {
...
+  /**
+   * sets track encoding that server should send to the client library.
+   * The encoding will be sent whenever it is available. If choosen encoding is
+   * temporarily unavailable, some other encoding will be sent until choosen encoding
+   *  becomes active again.
+   *
+   * @param trackId id of a track which encoding you want to select
+   * @param encoding encoding to select
+   */
+  const setTargetTrackEncoding = useCallback(
```
Comment (graszka22) [inline src/index.tsx:557]: this one is not related to the local video track, so it would be better to move it to separate function?

**Review by graszka22 — CHANGES_REQUESTED** [review]

File: src/index.tsx (hunk 1 above)
Comment (skyman503) [reply src/index.tsx:557]: I placed it in a new hook.

File: src/index.tsx [hunk 2]
```diff
@@ -518,11 +518,46 @@ export function useEndpoints() {
...
```
Comment (graszka22) [inline src/index.tsx:534]: no need for a hook, just function is enough 
see for example `updateEndpointMetadata` function

**Review by graszka22 — CHANGES_REQUESTED** [review]

**Review by graszka22 — CHANGES_REQUESTED** [review]
also `setVideoTrackBandwidth` is related to the camera track, we can also put it in the `useCamera` hook

File: src/__mocks__/index.tsx [hunk 3]
```diff
@@ -77,6 +81,7 @@ export const useScreencast = () => {
     simulcastConfig: NOOP,
     setScreencastTrackEncodingBandwidth: NOOP,
     setScreencastTrackBandwidth: NOOP,
+    setVideoTrackBandwidth: NOOP,
```
Comment (graszka22) [inline src/__mocks__/index.tsx:84]: wait, it's not in useScreeencast 🤔

**Review by graszka22 — CHANGES_REQUESTED** [review]

**Review by graszka22 — APPROVED** [review]

File: src/__mocks__/index.tsx (hunk 3 above)
Comment (skyman503) [reply src/__mocks__/index.tsx]: yeah, good catch, I always mix up simulcast and screencast
